cmd/nginx-ingress/main.go

```diff
@@ -388,6 +392,7 @@ func main() {
 		NginxStatusPort:                *nginxStatusPort,
 		StubStatusOverUnixSocketForOSS: *enablePrometheusMetrics,
 		TLSPassthrough:                 *enableTLSPassthrough,
+		SpiffeCerts:                    *spireAgentAddress != "",
```

Comment:
should this be `SpireCerts`?

Reply:
I think `SpiffeCerts` is correct, see my comment below about `SVIDs`